### ISSUE TITLE
Bazzite-deck image and instructions not working

### ISSUE THREAD

**Issue by morissonmaciel** [ublue-os/bazzite#1030]
### Describe the bug

Selecting from Bazzite.gg PC Desktop, AMD and Seam Game Mode, provides me bazzite-deck.iso which, flashed using balenaEtcher, only shows initial boot options screen. Choosing installing Bazzite nothing happens.

### What did you expect to happen?

Choosing installing Bazzite or any other options would start interactive screen with user (me).

### Output of `rpm-ostree status`

```shell
Could not run this command. Trying to install.
```

### Hardware

Ryzen 5 7600
RX 7800 XT
M2 1TB SSD
32 GB DDR5 6000Mhz (expo) RAM

**Comment by KyleGospo** [comment]
See if the normal Fedora Silverblue or Kinoite installers work for you. If they do you can rebase to Bazzite with the instructions on our website. 

If they don't, make sure you dont have legacy bios emulation turned on.

**Comment by morissonmaciel** [comment]
Hello again! 
@KyleGospo I managed to install both ChimeraOS and Windows 11 in the same system without problems. Haven't tried Fedora, although.
After an entire Sunday trying to install it, this is my update:

It seems the bazzite-deck image failed to initram with the shipped kernel due to a conflict when both iGPU and dGPU are enabled in bios. When force disabling iGPU, the installation process continues normally.

Tried to install it in another rig with a Ryzen 7700 CPU (iGPU also) and RX 7900 XTX and the same problem occurred.

In the end, I needed to uninstall Bazite OS 3.0 cause something was still wrong with Gamescope + AMD GPU with shipped packages, producing this following rainbow image on my TV.

![IMG_0004](https://github.com/ublue-os/bazzite/assets/11509104/6e600e14-b74d-4114-b383-3e1380426340)

I had haven't any issues with Chimera, Arch, or Windows at the same rig (with RX 7800 XT).

Thanks for your support, in advance.
